src/GregClient/Requests/BanPackage.cs

```diff
@@ -0,0 +1,37 @@
+﻿using RestSharp;
+
+namespace Greg.Requests
+{
+    public class BanPackage : Request
+    {
+        private enum Action { Ban, Unban }
+        private readonly Action banAction;
+        private readonly string packageId;
+
+        public BanPackage(string packageId, bool banPackage)
```

Comment:
Let's not use the `bool` but instead use your nice `BanPackage.Action` enum as the parameter.  `bool`'s are generally hard to understand when used like this.